apps/gallery/js/MetadataParser.js

```diff
@@ -500,60 +586,67 @@ var metadataParser = (function() {
         canvas.toBlob(function(blob) {
```

Comment:
Nit: Got it after looking at createThumbnailAndPreview call Line 352 and definition on Line 499 that we are calling metadataerror callback here, I guess for readability, it would have helped to keep names to metadataCallback, metadataError here same as in createJPEGThumbnailAndPreview.